Patch created by brosx "diffPackage" tool

usage:
    diffPackage <tarName> <pkgName> <outName> <patchName> <files>

actual:
    diffPackage "libcap-2.25.tar.xz" "libcap" "libcap" "include-packaged-kernel-headers-when-building-host-tool" "\
        libcap/Makefile \
    ";


===============================================================================
--- a/libcap/Makefile	2016-01-31 01:01:41.000000000 +0100
+++ b/libcap/Makefile	2018-01-13 18:30:55.000000000 +0100
@@ -35,7 +35,7 @@
 		$< >$@
 
 _makenames: _makenames.c cap_names.list.h
-	$(BUILD_CC) $(BUILD_CFLAGS) $< -o $@
+	$(BUILD_CC) $(BUILD_CFLAGS) -I $(KERNEL_HEADERS) $< -o $@
 
 cap_names.h: _makenames
 	./_makenames > cap_names.h
